qualcommax: ipq807x: Create working factory tar for WAX620 and WAX630.

ServError opened the pull request:
Create factory tar for WAX620 and WAX630 which is accepted by the stock UI.
Must use 'Boot up Backup Firmware' button on stock upgrade page after install and reboot to swap partitions.

skramstad reviewed (approved):
LGTM.

musashino205 commented on target/linux/qualcommax/image/ipq807x.mk:
```diff
@@ -326,6 +326,10 @@ define Device/netgear_wax620
 	BLOCKSIZE := 128k
 	PAGESIZE := 2048
 	SOC := ipq8072
+ifneq ($(CONFIG_TARGET_ROOTFS_INITRAMFS),)
```
Is this required? No recipe based on initramfs image is found in `IMAGE/ui-factory.tar`.

ServError replied:
It is not. Updated, thank you.

ServError committed:
qualcommax: ipq807x: Create working factory tar for WAX620 and WAX630.

Create factory tar for WAX620 and WAX630 which is accepted by the stock UI.
Must use 'Boot up Backup Firmware' button on stock upgrade page after install and reboot to swap partitions.

Signed-off-by: Erik Servili <serverror@serverror.com>
Link: https://github.com/openwrt/openwrt/pull/18334
Signed-off-by: Robert Marko <robimarko@gmail.com>

robimarko commented:
Thanks! Rebased on top of main and merged!

lymanepp commented:
@ServError, I've successfully installed this on my two WAX620s. Thanks for making this doable without cracking them open!
